analyzers/its/expected/akka.net/Akka--netstandard2.0-S2222.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S2222",
+"message":  "Unlock this lock along all executions paths of this method.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka\Actor\RepointableActorRef.cs",
```

Comment:
I was hoping for a first TP in an external project. Not yet :)

Reply:
Not yet 😒 But this is a bug rule, there's a high chance that the production code already suffered from it in the past and it's therefore fixed. Would be interesting to find TP on Peach one day.